txpipe/twopoint.py

```diff
@@ -564,7 +565,113 @@ def read_metadata(self):
 
         return meta
 
-    
+# Aperture Mass class that inherits from TXTwoPoint
+class TXApertureMass(TXTwoPoint):
+
+    name='TXApertureMass'
+    inputs = [
+        ('calibrated_shear_catalog', ShearCatalog),
+        ('calibrated_lens_catalog', HDFFile),
+        ('shear_photoz_stack', HDFFile),
+        ('patch_centers', TextFile),
+        ('tracer_metadata', HDFFile),
+    ]
+    outputs = [
+        ('aperturemass_data', SACCFile),
```

Comment:
Sorry to be finickity but could you put an underscore between aperture and mass, here and below?

Reply:
Done